bedrock/foundation/templates/foundation/annualreport/2020/index.html

```diff
@@ -53,36 +53,39 @@ <h2><span class="highlight dark">Our impact</span></h2>
   <div class="mzp-l-content leadership no-js-hidden">
     <h2 class="leadership-intro">A word<br> <strong>from our leadership</strong></h2>
     <div class="mzp-l-card-third">
-      <div class="has-modal" data_modal_id="mitchell-baker-letter">
+
       {{ card(
         title='Mitchell Baker',
         ga_title='Mitchell Baker',
         tag_label='Steering Committee',
         desc='Chief Executive Officer, Chairwoman of Mozilla Foundation',
+        class='has-modal',
+        attributes="data-modal-id=mitchell-baker-letter",
```

Comment:
Here's an example of `attributes` being used here. The main attribute I needed to add so the modals would function properly was `data-modal-id`. Because it's a one-off attribute to this page, it made more sense to just create `attributes` and manipulate it to be used anytime we need to implement a unique attribute to a protocol macro.